### PR TITLE
Using new mesh interface for properties in class ElementExtraction

### PR DESCRIPTION
**Pull request by TomFischer**
as titled.

### CODE REVIEW

File: MeshLib/MeshEditing/ElementExtraction.h [hunk 1]
```diff
@@ -57,8 +57,9 @@ class ElementExtraction
 	/// Updates the vector of marked elements with values from vec.
 	void updateUnion(const std::vector<std::size_t> &vec);
 
-	/// Removes elements from vec_removed in vec_src_elems
-	std::vector<MeshLib::Element*> excludeElements(const std::vector<MeshLib::Element*> & vec_src_elems, const std::vector<std::size_t> &vec_removed) const;
+	std::vector<MeshLib::Element*> excludeElementCopy(
```
Comment (endJunction) [inline MeshLib/MeshEditing/ElementExtraction.h]: documentation, because the name is not selfdescribing...

**Comment by endJunction** [comment]
OK, but documentation.

File: MeshLib/MeshEditing/ElementExtraction.h (hunk 1 above)
Comment (TomFischer) [inline MeshLib/MeshEditing/ElementExtraction.h]: Added docu.

**Comment by endJunction** [comment]
:fast_forward:

**Pull request merged**